HTML/Rebooting M-OS.html

```diff
@@ -272,25 +272,25 @@ <h2>Calculating Value A</h2>
                             </th>
                         </tr>
                     </table>
-                    <p style="font-size: 14.5px"><i>T - Square moves from bottom to top.</i></p>
-                    <p style="font-size: 14.5px"><i>B - Square moves from top to bottom.</i></p>
-                    <p style="font-size: 14.5px"><i>R - Square moves from left to right.</i></p>
-                    <p style="font-size: 14.5px"><i>L - Square moves from right to left.</i></p>
-                    <p style="font-size: 14.5px"><i>BL - Square moves from top-right corner to botton-left corner.</i></p>
-                    <p style="font-size: 14.5px"><i>TR - Square moves from botom-left corner to top-right corner.</i></p>
-                    <p style="font-size: 14.5px"><i>BR - Square moves from top-left corner to bottom-right corner.</i></p>
-                    <p style="font-size: 14.5px"><i>TL - Square moves from bottom-right corner to top-left corner.</i></p>
+                    <p><i>T - Square moves from bottom to top.</i></p>
+                    <p><i>B - Square moves from top to bottom.</i></p>
+                    <p><i>R - Square moves from left to right.</i></p>
+                    <p><i>L - Square moves from right to left.</i></p>
+                    <p><i>BL - Square moves from top-right corner to botton-left corner.</i></p>
+                    <p><i>TR - Square moves from botom-left corner to top-right corner.</i></p>
+                    <p><i>BR - Square moves from top-left corner to bottom-right corner.</i></p>
+                    <p><i>TL - Square moves from bottom-right corner to top-left corner.</i></p>
                 </div>
                 <div class="page-footer relative-footer">Page 1 of 4</div>
             </div>
             <div class="page page-bg-02">
                 <div class="page-header">
                     <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
-                    <span class="page-header-section-title">Rebooting M-OS</span>
+                    <span class="page-header-section-title">Rebooting Mystery OS</span>
                 </div>
                 <div class="page-content">
                     <h2>Calculating Value B</h2>
-                    <p>For this you need to modify your A value. Apply to value A 4 operations based on the square's colors in the order they appear on it. This is your value B.</p>
+                    <p>For this, you need modify your value A. Apply to value A 4 operations based on square's colors in order, they appears on it. This is your value B. </p>
```

Comment:
You're trying to revert all grammatical errors.

Reply:
huh, i just updated it with the new one provided in request.